client/src/Tools/_framework/Paths/Activities.jsx

```diff
@@ -443,15 +476,17 @@ export function Activities() {
                       position,
                       folder.content.length,
                     )}
+                    suppressAvatar={true}
+                    showOwnerName={false}
                     imageLink={
                       activity.isFolder
                         ? `/activities/${activity.ownerId}${activity.id ? "/" + activity.id : ""}`
```

Comment:
If `activity` exists, then `activity.id` must exist, so you don't need to check for its existence. I believe this should work for the `imageLink` if `isForlder` is `true`.
```
`/activities/${activity.ownerId}/${activity.id}`
```